Flow.Launcher/App.xaml.cs

```diff
@@ -71,9 +70,16 @@ private async void OnStartupAsync(object sender, StartupEventArgs e)
                 PluginManager.LoadPlugins(_settings.PluginSettings);
                 _mainVM = new MainViewModel(_settings);
                 API = new PublicAPIInstance(_settingsVM, _mainVM, _alphabet);
+
+                Http._api = API;
+                Http.Proxy = _settings.Proxy;
+                
                 await PluginManager.InitializePlugins(API);
                 var window = new MainWindow(_settings, _mainVM);
 
+                
+
+
```

Comment:
lol you love your spaces dont you 😛

Reply:
ah 🤣🤣🤣 I love to separate different parts of code doing different with spaces🤣🤣🤣